packages/lesswrong/components/review/ReviewVoteTableRow.tsx

```diff
@@ -177,7 +186,7 @@ const ReviewVoteTableRow = (
         <PostsItemComments
           small={false}
           commentCount={postGetCommentCount(post)}
-          unreadComments={post.lastVisitedAt < post.lastCommentedAt}
+          unreadComments={(post.lastVisitedAt < (markedVisitedAt || post.lastCommentedAt))}
```

Comment:
```suggestion
          unreadComments={(markedVisitedAt || post.lastVisitedAt) < post.lastCommentedAt}
```
The code works as it was written, but is not logically what you wanted.